packages/website/pages/tokens.js

```diff
@@ -68,20 +68,24 @@ const TokenRow = ({ token, index, copied, onCopy, deleting, onDelete }) => {
   const sharedArgs = { index }
   return (
     <tr>
-      <TableElement {...sharedArgs} important>{token.name}</TableElement>
+      <TableElement {...sharedArgs} important breakAll={false}>
+        {token.name}
+      </TableElement>
       <TableElement {...sharedArgs} important>
-        <code style={{ wordWrap: 'break-word' }}>{token.secret}</code>
+        <code style={{ whiteSpace: 'nowrap' }}>
+          {token.secret.substr(0, 20)}...{token.secret.substr(token.secret.length - 20, token.secret.length)}
```

Comment:
Could we move the `20` to a computed property of the component?
Easier to use and tweak this value.